internal/driver/glfw/window.go

```diff
@@ -582,12 +592,25 @@ func (w *window) mouseMoved(viewport *glfw.Window, xpos float64, ypos float64) {
 
 	if w.cursor != cursor {
 		// cursor has changed, store new cursor and apply change via glfw
+		newCursor, isStdCursor := fyneToNativeCursor(cursor)
```

Comment:
I suggest to invert the meaning of the `bool` returned by `fyneToNativeCursor(…)` to `isCustomCursor`.
It makes `false` the more common value and emphasises that there is something special in case it is `true`.

Reply:
That makes sense, thanks!